functions/ocr/app/ocr_test.go

```diff
@@ -0,0 +1,131 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package ocr
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"fmt"
+	"strings"
+	"testing"
+
+	"cloud.google.com/go/storage"
+	"github.com/GoogleCloudPlatform/golang-samples/internal/testutil"
+	"golang.org/x/text/language"
+	pubsubpb "google.golang.org/genproto/googleapis/pubsub/v1"
+)
+
+const (
+	menuName        = "menu.jpg"
+	signName        = "sign.png"
```

Comment:
Do these files need to be checked in? Do Python and Java use a common bucket we can reuse here? In general, it's good to avoid checking binary files into version control.

Reply:
The other version have an images folder, but since the files are also in cloud storage they don't need to be in the local directory. Should I remove them?

Reply:
Yes, please remove them.

Reply:
Done